development/tools/cmd/gcrcleaner/README.md

```diff
@@ -0,0 +1,46 @@
+# GCR cleaner
+
+## Overview
+
+This command finds and removes old GCR images created by jobs in the Google Cloud Platform (GCP) project.
+
+There are three conditions used to find images for removal:
+- The repository name pattern is not on the ignored list.
+- The `creationTimestamp` value of the images, that is used to find addresses, exists at least for a preconfigured number of hours.
+
+GCR images that meet these conditions are subject to removal.
+
+## Usage
+
+For safety reasons, the dry-run mode is the default one.
+To run it, use:
+```bash
+env GOOGLE_APPLICATION_CREDENTIALS={path to service account file} go run main.go \
+    --repository={gcloud repository url}
+```
+
+To turn the dry-run mode off, use:
+```bash
+env GOOGLE_APPLICATION_CREDENTIALS={path to service account file} go run main.go \
+    --repository={gcloud repository url} \
+    --dry-run=false
+```
+
+## Flags
```

Comment:
In some documents, `## Flags` are formatted as [this](https://github.com/kyma-project/test-infra/tree/main/development/image-syncer#flags), but I didn't suggest it here, 'cause you also have the **Required** fields.